src/test/testFixtureUtils.tsx

```diff
@@ -215,34 +227,43 @@ export async function loadFixtureData(testName: string): Promise<UIStore> {
   return store;
 }
 
-export function sendLoadedRegionsToMockEnvironment(value: loadedRegions): void {
-  window.mockEnvironment?.sendSocketMessage(
-    JSON.stringify({
-      method: "Session.loadedRegions",
-      params: value,
-      sessionId: DEFAULT_SESSION_ID,
-    })
-  );
-}
-
-export function sendMessageToMockEnvironment(value: Message): void {
-  window.mockEnvironment?.sendSocketMessage(
-    JSON.stringify({
-      method: "Console.newMessage",
-      params: {
-        message: value,
-        sessionId: DEFAULT_SESSION_ID,
-      },
-    })
-  );
-}
+type Tuple = LoadedRegionsTuple | MessageTuple | SourceTuple;
 
-export function sendSourceToMockEnvironment(value: newSource): void {
-  window.mockEnvironment?.sendSocketMessage(
-    JSON.stringify({
-      method: "Debugger.newSource",
-      params: value,
-      sessionId: DEFAULT_SESSION_ID,
-    })
-  );
+export function sendValuesToMockEnvironment(...values: Array<Tuple>): void {
+  values.forEach(([method, value]) => {
+    switch (method) {
+      case "Session.loadedRegions":
+        window.mockEnvironment?.sendSocketMessage(
+          JSON.stringify({
+            method,
+            params: value,
+            sessionId: DEFAULT_SESSION_ID,
+          })
+        );
+        break;
+      case "Console.newMessage":
+        window.mockEnvironment?.sendSocketMessage(
+          JSON.stringify({
+            method,
+            params: {
```

Comment:
:question: I notice a difference in `params, sessionId` between these different messages. Is that correct?  (If it is, it bothers my sense of symmetry :)  )

Reply:
Nope, you're right. `sessionId` should be outside of `params`. Doesn't seem to affect anything though but I'll fix.